data/campaigns/Under_the_Burning_Suns/utils/macros.cfg

```diff
@@ -136,7 +136,7 @@ if(red <= green and green <= blue + 3, ({NEWCHAN} / 255.0)^2 * 255.0, {ORIGCHAN}
 #enddef
 
 #define UTBS_XP_MODIFIER
-    experience_modifier={ON_DIFFICULTY 100 150 200}
+    experience_modifier={ON_DIFFICULTY 100 125 150}
```

Comment:
This balance change should be in a separate commit imo, along with the recall cost change below.

Reply:
Probably.